schemas/view-schema.json

```diff
@@ -107,7 +107,7 @@
             },
             "custom_timings": {
               "type": "object",
-              "description": "User custom timings of the view",
+              "description": "User custom timings of the view, as timing name is used as facet path, it must contain only contains letters, digits, or the characters - _ . @ $",
```

Comment:
```suggestion
              "description": "User custom timings of the view. As timing name is used as facet path, it must only contains letters, digits, or the characters - _ . @ $",
```

Reply:
thx, I've kept the same phrasing as in the error message in the UI